files/run.sh

```diff
@@ -11,6 +11,16 @@ rsync -a /inventory.generics/ /inventory.pre/
 rsync -a /extra/ //inventory.pre/
 rsync -a /opt/configuration/inventory/ /inventory.pre/
 
+# get version files from /interface/versions
+
+cp /interface/versions/osism-ansible.yml /inventory.pre/group_vars/all/100-versions-osism-ansible.yml
```

Comment:
This leaves the impression that it is meant to be ordered after 99-* but I think it will be between 10 and 11. Better stick to two-digit numbers for consistency.

Reply:
In ansible-defaults we use 3 digits. 001 - 999. Therefore, it should work if we use the 100.